### ISSUE TITLE
Publish the docker image

### ISSUE THREAD

**Issue by azukaar** [abdeladim-s/subsai#144]
Not having the image publish sort of defeat the purpose of having a docker image, it would be nice to have it published to docker hub (you can have a github CI script to auto publish on commit)

**Comment by abdeladim-s** [comment]
@azukaar, Yes, having the image pre-built is great. I've wanted to do this for a while but haven't had the chance.
I've quickly created a GitHub action and pushed the image to Docker Hub. You can find it [here]( https://hub.docker.com/r/absadiki/subsai/).
Hope this helps.

**Comment by azukaar** [comment]
amazing thanks

**Comment by abdeladim-s** [comment]
I just followed and used the action from the [Github docs](https://docs.github.com/en/actions/use-cases-and-examples/publishing-packages/publishing-docker-images). 
I believe `main` refers to the main branch and is set automatically [here](https://github.com/abdeladim-s/subsai/blob/6946886a57d4e610d65945baa578018db069de90/.github/workflows/docker-image.yml#L40). 
I think the problem is that metadata-action only sets the latest tag if the main branch is called `master` and I need to add a hack to make it work with `main`.  Will look into that!
